### DISCUSSION TITLE
I want to investigate on excessive self-deal and datacap abuse case

### DISCUSSION THREAD

**Discussion by boramida** [filecoin-project/notary-governance#488] [General]
I want to investigate on excessive self-deal and datacap abuse case.
https://filfox.info/en/address/f01675012
https://filfox.info/en/address/f01759195
https://filfox.info/en/address/f01778041
These SPs are power up about 10PiB in very short time.
They get deals from client f1l2hkjqfh5abpd5x6lxj4tjrphaa44pnumt5ms5i only.
Most normal SPs has been very hardly worked for veriefied deal in estury and Fil+.
But few deal is successful.
But I think these cases are unfair for DataCap.

**Comment by cryptowhizzard** [comment]
Hello @boramida 

I have referenced this issue here for you for investigation.

https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/153
